lib/galaxy/webapps/galaxy/script.py

```diff
@@ -12,16 +12,15 @@ def main() -> None:
     parser.add_argument("--config", "-c", help="Galaxy config file")
     parser.add_argument("--single-user", "-s", help="Single user mode as user SINGLE_USER")
     parser.add_argument("--bind", "-b", default="localhost:8080", help="Bind to <host>:<port>")
-    parser.add_argument(
-        "--client-path", "-n", default="node_modules/@galaxyproject/galaxy-client", help="Path to Galaxy client"
-    )
+    parser.add_argument("--client-path", "-n", help="Path to Galaxy client")
```

Comment:
I would perhaps drop this script. IMO we shouldn't have that many entrypoints, it's another impediment to deploying galaxy with celery, and the transition from this to a setup where you first run a setup command seems like unnecessary complexity.

Essentially I would feel uncomfortable recommending this entrypoint to anyone (given the lack of non-dev options), so I am not sure who this would serve ?

Reply:
It was sort of a proof of concept of a single command to start a Galaxy webserver with no config. I'm working on Gravity changes to do the same, though. I'll leave it for now but not advertise it, we can scrap it later.

Reply:
I think gravity is the right place to set the 10 - 20  things you might want to set in gunicorn / uvicorn / hypercorn / granian and all the other servers that can run Galaxy. Those aren't Galaxy config options and it'd be weird to have that be done in Galaxy IMO.